### PR TITLE
[DOCS] 7.15.1 Release Notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses #1142.

Preview [here](https://security-docs_1146.docs-preview.app.elstc.co/guide/en/security/7.x/release-notes.html).

### CODE REVIEW

File: docs/release-notes/7.15.asciidoc [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+== 7.15
+
+[discrete]
+[[release-notes-7.15.1]]
+=== 7.15.1
+
+[discrete]
+[[bug-fixes-7.15.1]]
+==== Bug fixes and enhancements
+* Supports newlines in the JIRA summary field. Within JIRA, newlines are replaced with commas ({pull}113571[#113571]).
```
Comment (jmikell821) [inline docs/release-notes/7.15.asciidoc:13]: Should this be `new lines` instead?

```suggestion
* Supports new lines in the JIRA summary field. New lines are replaced with commas ({pull}113571[#113571]).
```

File: docs/release-notes/7.15.asciidoc [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+* Resolves a bug that prevented the Endpoints table from reseting after the KQL query was removed ({pull}112595[#112595]).
```
Comment (jmikell821) [inline docs/release-notes/7.15.asciidoc:14]:
```suggestion
* Fixes a bug that prevented the Endpoints table from resetting after the KQL query was removed ({pull}112595[#112595]).
```

**Review by jmikell821 — COMMENTED** [review]
Just a few small comments.

**Comment by nastasha-solomon** [comment]
@mergify backport 7.15

**Pull request merged**